client/v3/naming/resolver/resolver.go

```diff
@@ -31,9 +32,14 @@ type builder struct {
 }
 
 func (b builder) Build(target gresolver.Target, cc gresolver.ClientConn, opts gresolver.BuildOptions) (gresolver.Resolver, error) {
+	endpoint := target.URL.Path
```

Comment:
Could you add a comment above line 35 something like below?

```
Refer to https://github.com/grpc/grpc-go/blob/16d3df80f029f57cff5458f1d6da6aedbc23545d/clientconn.go#L1587-L1611
```

Reply:
Done, thank you for the review.